docs/security/seccomp.md

```diff
@@ -81,3 +80,65 @@ profile.
 $ docker run --rm -it --security-opt seccomp:unconfined debian:jessie \
     unshare --map-root-user --user sh -c whoami
 ```
+
+### Syscalls blocked by the default profile
+
+Below is a list of certain syscalls and why we are blocking them in the default
```

Comment:
```
Docker's default seccomp profile is a whitelist which specifies the calls that
are allowed. The table below lists the significant (but not all) syscalls
blocked as a result of this whitelist. The table includes the reason each
syscall is blocked rather than white-listed
``